importer/tasks.py

```diff
@@ -385,6 +387,7 @@ def get_asset_urls_from_item_resources(resources):
     """
 
     assets = []
+    item_resource_url = resources[0]["url"] or ""
```

Comment:
Do we have to worry about getting more than one of these? It seems like we might want to at least log if there are discrepancies

Reply:
I'm not sure, actually. I was hoping to have a chance to test this with the real spreadsheet in dev / test.